test/routes/parcels.test.js

```diff
@@ -0,0 +1,86 @@
+let parcelsService
+const parcel = { ref: 'SD75492628' }
+function createMocks () {
+  jest.mock('../../server/services/parcels-service')
+  parcelsService = require('../../server/services/parcels-service')
+  parcelsService.getParcels = () => { return Promise.resolve([parcel]) }
+}
+
+function extractSessionCookie (response) {
+  const setCookie = response.headers['set-cookie']
+  return (setCookie && setCookie[0]) ? setCookie[0].split(';')[0] : ''
+}
+
+function getRedirectOptions (response) {
+  const cookie = extractSessionCookie(response)
+  return {
+    method: 'GET',
+    headers: { cookie },
+    url: response.headers.location
+  }
+}
+
+describe('Parcels route test', () => {
+  let createServer
+  let server
+
+  beforeAll(() => {
+    createMocks()
+    createServer = require('../../server/createServer')
+  })
+
+  beforeEach(async () => {
+    server = await createServer()
+    await server.initialize()
+  })
+  test('GET / route returns 200', async () => {
```

Comment:
Should this be 'GET /parcels route returns 200'

Reply:
indeed it should, thanks I'll get it changed